test/server/script-processor-test.js

```diff
@@ -156,6 +157,16 @@ describe('Script processor', function () {
         expect(isScriptProcessed(processed)).to.be.true;
     });
 
+    it('Should add strict mode directive', function () {
```

Comment:
The test name is not clear.
Something like 'Should add strict mode directive after top hammerhead processing header  comment'